docs/publish.rst

```diff
@@ -67,6 +69,10 @@ the options are:
 * ``wheels_macos``: binary wheels for MacOS X
 * ``wheels_windows``: binary wheels for Windows
 
+Finally the ``libraries`` section allow you to pre-install libraries that are required to install the ``sdist`` build.
+Since this is done in order to test that the built source distribution.
+Further, the ``sdist`` build uses the horrible and outdated Ubuntu distribution, everything is installed using the ``apt`` package manager.
```

Comment:
```suggestion
This can be needed in order to test the built source distribution.
The libraries are installed using the ``apt`` package manager.
```